sdk/graphql/forta.ts

```diff
@@ -0,0 +1,145 @@
+                ) {
+                    alerts(input:{
+                        bots: $bots,
+                        addresses: $addresses,
+                        after: $after,
+                        alertId: $alertId,
+                        chainId: $chainId,
+                        projectId: $projectId,
+                        scanNodeConfirmations: $scanNodeConfirmations,
+                        severities: $severities,
+                        transactionHash: $transactionHash,
+                        blockSortDirection: $blockSortDirection,
+                        first: $first,
+                        createdSince: $createdSince,
+                        blockDateRange: $blockDateRange,
+                        blockNumberRange: $blockNumberRange
+                    }) {
+                        alerts {
+                            alertId
+                            addresses
+                            contracts {
+                                address
+                                name
+                                projectId
+                            }
+                            createdAt
+                            description
+                            findingType
+                            name
+                            projects {
+                                id
+                            }
+                            protocol
+                            scanNodeCount
+                            severity
+                            source {
+                                transactionHash
+                                bot {
+                                    id
+                                }
+                            }
+                        }
+                        pageInfo {
+                            hasNextPage
+                            endCursor {
+                                alertId
+                                blockNumber
+                            }
+                        }
+                    }
+            }
+        `,
+        "variables": {
+            bots: options.botIds,
+            addresses: options.addresses,
+            after: options.startingCursor,
+            alertId: options.alertId,
+            chainId: options.chainId,
+            first: options.first,
+            projectId: options.projectId,
+            scanNodeConfirmations: options.scanNodeConfirmations,
+            severities: options.severities,
+            transactionHash: options.transactionHash,
+            blockSortDirection: options.blockSortDirection,
+            createdSince: options.createdSince?.getDate(),
+            blockDateRange: options.blockDateRange ? 
+                { startDate: options.blockDateRange.startDate.toISOString().split('T')[0], endDate: options.blockDateRange.endDate.toISOString().split('T')[0]} :
+                undefined,
+            blockNumberRange: options.blockNumberRange
+        } 
```

Comment:
btw alerts can also be filtered by 
```
alertName
blockTimestampRange
```
for more details about all the possible query input fields - https://docs.forta.network/en/latest/forta-api-reference/#definition-AlertsInput